src/xpk/parser/cluster.py

```diff
@@ -449,6 +449,21 @@ def add_shared_cluster_create_optional_arguments(args_parsers):
             ' --custom-tpu-nodepool-arguments="--enable-ip-alias"'
         ),
     )
+    custom_parser.add_argument(
+        '--enable-workload-identity',
```

Comment:
One discussion we can think through is if we want to support workload-identity separately from gcsfuse. Do we see a need to support workload-identity by itself such as the additional dev support of the feature by itself in XPK? The alternative is we just support the current request (from what I am aware) is just gcsfuse-csi-driver support. If we don't currently see value for `--enable-workload-identity` flag by itself, we don't need to add it now (one way door)! Curious what you think!

Reply:
That's what I'm waiting for the decision. For now it's possible to only provide only --enable-gcsfuse-csi-driver flag which will enable WI, too.
We can remove the WI flag later if we no need for that. What do you think?

Reply:
> We can remove the WI flag later if we no need for that. What do you think?

Yes I agree we can remove it, though I would prefer to remove it now since it is harder to remove later! But I'll leave the decision up to you here :)

Reply:
I'll leave it to @PBundyra as we discussed and close this PR.